doc/design/libsvm_format_column.md

```diff
@@ -0,0 +1,112 @@
+# Support LibSVM format on SQLFlow
+
+## Background
+
+SQLFlow extends SQL syntax to support End-to-End machine learning pipeline. 
+However, due to the variety of the data format (like CSV, LibSVM, etc.), 
```

Comment:
libsvm is not a file format. According to https://www.csie.ntu.edu.tw/~cjlin/libsvm/, LIBSVM is "A Library for Support Vector Machines". What do you really want to say here?

Reply:
I have added the `Definition` section to make it more clear. Please see [here](https://github.com/sql-machine-learning/sqlflow/pull/2554/files#diff-f73c2f4f93fc028434d604606630b34dR3).